Source/JavaScriptCore/runtime/JSString.h

```diff
@@ -262,6 +262,9 @@ class JSString : public JSCell {
 
     bool is8Bit() const;
 
+    JSString* replaceOneCharImpl(JSGlobalObject*, UChar search, JSString* replacement, bool& found, unsigned recursionLimit, uint8_t* stackLimit);
```

Comment:
This shouldn't be public and typically out parameters are at the end of the arguments list.